indico/modules/categories/controllers/management.py

```diff
@@ -58,8 +60,22 @@ def _category_query_options(self):
         return children_strategy,
 
     def _process(self):
+        show_all_events = request.args.get('show_all') == '1'
+        page_number = int(request.args.get('page', 1))
+        order_columns = {'start_dt': Event.start_dt, 'title': db.func.lower(Event.title)}
+        direction = db.desc if request.args.get('desc', '1') == '1' else db.asc
+        order_column = order_columns[request.args.get('order', 'start_dt')]
```

Comment:
I'd catch KeyErrors from this whole block and redirect without a page/order in that case